algorithms/persian.sbl

```diff
@@ -115,34 +112,34 @@ define Suffix_Adjective as (
 
 define Exception as (
     [substring] among (
-        '{sin}{ye}'
-        '{dal}{re}'
-        '{alef}{ye}{nun}'
-        '{alef}{ye}'
-        '{be}{re}'
-        '{nun}{alef}{mim}'
-        '{pe}{sin}'
-        '{alef}{sin}{te}{alef}{nun}'
-        '{be}{re}{alef}{ye}'
-        '{pe}{ye}{vav}{nun}{dal}'
-        '{mim}{ye}{lam}{alef}{dal}{ye}'
-        '{mim}{lam}{ye}'
-        '{che}{nun}{dal}'
-        '{be}{re}{khe}{ye}'
-        '{he}{mim}{ye}{nun}'
-        '{he}{mim}{alef}{nun}'
-        '{he}{mim}{he}'
-        '{mim}{te}{re}'
-        '{te}{ye}{mim}'
-        '{sin}{lam}{alef}{mim}'
-        '{alef}{sin}{lam}{alef}{mim}'
-        '{ye}{ain}{nun}{ye}'
-        '{aa}{lam}{be}{vav}{mim}'
+        '{sin}{ye}' ()
+        '{dal}{re}' ()
+        '{alef}{ye}{nun}' ()
+        '{alef}{ye}' ()
+        '{be}{re}' ()
+        '{nun}{alef}{mim}' ()
+        '{pe}{sin}' ()
+        '{alef}{sin}{te}{alef}{nun}' ()
+        '{be}{re}{alef}{ye}' ()
+        '{pe}{ye}{vav}{nun}{dal}' ()
+        '{mim}{ye}{lam}{alef}{dal}{ye}' ()
+        '{mim}{lam}{ye}' ()
+        '{che}{nun}{dal}' ()
+        '{be}{re}{khe}{ye}' ()
+        '{he}{mim}{ye}{nun}' ()
+        '{he}{mim}{alef}{nun}' ()
+        '{he}{mim}{he}' ()
+        '{mim}{te}{re}' ()
+        '{te}{ye}{mim}' ()
+        '{sin}{lam}{alef}{mim}' ()
+        '{alef}{sin}{lam}{alef}{mim}' ()
+        '{ye}{ain}{nun}{ye}' ()
+        '{aa}{lam}{be}{vav}{mim}' ()
     )
 )
 
 define stem as (
-    Exception
+    ( Exception )
```

Comment:
This means the same with or without the parentheses (just noting as the commit that changed this was "fix exception section"...)